restlight-jaxrs-provider/src/main/java/io/esastack/restlight/jaxrs/adapter/MessageBodyWriterAdapter.java

```diff
@@ -69,10 +69,10 @@ public MessageBodyWriterAdapter(Providers providers) {
                 }
 
                 ResponseEntityStreamUtils.close(context);
-                return HandledValue.succeed(null);
+                return Result.ok();
             }
         }
-        return HandledValue.failed();
+        return Result.err();
```

Comment:
Err is a more general expression. In fact, if we really want to use a more accurate expression, failed is not very appropriate too. We should express it with unsupported. But for reuse, we reuse result.err(), instead of starting a new stove.in this case, there is no need to be too tangled about whether the expression is completely accurate, and the meaning is roughly the same.